samcli/commands/sync/command.py

```diff
@@ -62,7 +62,10 @@
 HELP_TEXT = """
 [Beta Feature] Update/Sync local artifacts to AWS
 
-By default, the sync command runs a full stack update. You can specify --code or --watch to switch modes
+By default, the sync command runs a full stack update. You can specify --code or --watch to switch modes.
+\b
+Sync also supports nested stacks and nested stack resources. For example
+$ sam sync --code --stack-name {stack} --resource-id ChildStack/ResourceId
```

Comment:
Should we use curly braces for `ChildStack` and `ResourceId` just like line 120?

Reply:
Probably a good idea